tests/acceptance/test_shared_issue.py

```diff
@@ -48,5 +48,7 @@ def test_cocoa_event(self):
 
         self.browser.get(u'/share/issue/{}/'.format(group.get_share_id()))
         self.browser.wait_until_not('.loading-indicator')
-        self.browser.wait_until('.entries')
+        self.browser.attempts(
```

Comment:
Is this any different to increasing the timeout?

Reply:
In other suites I've worked with the small amount of sleep and repeated DOM queries can help resolve synchronization problems in webdriver. But otherwise it is roughly equivalent to a longer timeout.

Reply:
That seems reasonable. Could you drop down the call timeout to 2 seconds so that we can determine whether this helps? Otherwise we can't tell whether waiting for 50 seconds is helping, or whether it's the retry.